src/components/navigation/verticalNav.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import { MenuIcon, XIcon } from "@heroicons/react/outline";
+import VerticalNavWrapper from "./verticalNavWrapper";
+
+const VerticalNav = ({ blok: { verticalNav }, className }) => {
+  const [navOpened, setNavOpened] = useState(false);
+  const toggleNav = () => {
+    setNavOpened(!navOpened);
+  };
+
+  let NavIcon = MenuIcon;
+  if (navOpened) {
+    NavIcon = XIcon;
+  }
+
+  return (
+    <nav
+      className={`${navOpened ? "su-shadow-xl" : ""} 
+      ${className}
+      su-relative su-mx-20 su-my-20`}
+      aria-label="Section Menu"
+    >
+      <button
+        type="button"
+        className="lg:su-hidden su-w-full su-flex su-justify-between su-font-semibold su-items-center su-mt-20"
```

Comment:
```suggestion
        className="su-group lg:su-hidden su-w-full su-flex su-justify-between su-font-semibold su-items-center su-mt-20"
```
To style the svg icon on button hover/focus, we need to assign the focusable element a `su-group` class. Otherwise the `group-hocus:` variant won't work in the icon. Please add other TW classes as needed to style this button.

Reply:
Hmm, it actually works fine without it, I added the remaining classes to make it as per design.

Reply:
Thanks for the styling changes. I ended up adding the su-group class anyway, there was a subtle delay between the hovering and color change without it (I think it default to currentColor which is inherited from the parent link styles, when relying on that it usually causes the slight delay in color change).